src/pages/Articles/Read/index.js

```diff
@@ -60,6 +66,36 @@ class Read extends Component {
     </div>
   );
 
+  renderArticleMenu = () => {
+    const { article } = this.props;
+    const user = getCurrentUser();
+    return (
+      user && (article.author.username === user.username)
+        ? null
+        : (
+          <div className="row">
+            <div className="col s1 offset-s11">
+              <a className="dropdown-trigger" data-target="report-article-dropdown">
+                <i className="material-icons right">more_vert</i>
+              </a>
+            </div>
+          </div>
+        )
+    );
+  };
+
+  renderArticle = (isFetched, article) => (
+    isFetched ? (
+      <ArticleViewLoader />
+    ) : (
+      <>
```

Comment:
Good work using React fragments syntax sugar but for readability replace this with `Fragment`

Reply:
I have refactored as advised, it now looks likes this:
```
renderArticle = (isFetched, article) => (
    isFetched ? (
      <ArticleViewLoader />
    ) : (
      <React.Fragment>
        <ArticleEditor {...this.props} readOnly />
        {this.renderTags(article.tags)}
        <LikeDislike slug={article.slug} />
      </React.Fragment>
    )
  );
```

Reply:
Okay this looks fine